### ISSUE TITLE
Packaging may fail when project uses <Import>

### ISSUE THREAD

**Issue by lanfeust69** [Azure/service-fabric-issues#41]
This is due to the implementation of the `GetPackageRootFile` task in `Microsoft.VisualStudio.Azure.Fabric.BuildTasks.dll`, which uses a stream to read project files. This loses the information about the location of the file, and breaks reading things such as `<Import Project="..\..\foo.targets" />` with a relative path, so that the path is resolved against the `.sfproj` file, which doesn't necessarily match.

Workaround is to use a `<UsingTask>` element in the `.sfproj`, with a copy of the implementation except for `FindPackageRootFilesDefinedInProject`, which becomes :
```csharp
  private IEnumerable<ITaskItem> FindPackageRootFilesDefinedInProject(string packageRootPath)
  {
    return ProjectCollection.GlobalProjectCollection.LoadProject(ProjectPath).Items
        .Where(i => i.ItemType == "None" || i.ItemType == "Content")
        .Select(i => new { Item = i, DestinationPath = GetDestinationPath(i, packageRootPath) })
        .Where(d => d.DestinationPath != null)
        .Select(d => CreateTaskItem(NormalizePath(d.Item.EvaluatedInclude), d.DestinationPath));
  }
```

**Comment by masnider** [comment]
Hi @lanfeust69 , there were some known bugs here that we think were fixed in our 5.2/2.2 Runtime & SDK update. If you update to the latest SDK & tools do you still see this issue?

**Comment by lanfeust69** [comment]
@masnider : Indeed, it worked after removal of the workaround (at least with 2.3, I think I also tested with 2.2, but I'm not quite sure).
